signalfx-tracing/signalfx-nodejs-tracing/hapi_pg/client.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+// Initialize the tracer and initiate auto-instrumentation
+// of all supported libraries and frameworks.  In this client example,
+// we instrumented the http module used by the contactKeeper client module.
+// const tracer = require('./contactKeeper/tracer');
+
+// Note that importing other modules should occur after init()
```

Comment:
Again here, there is no call to `init()` in sight, making this quite confusing

Reply:
Thanks! Would changing it to:
>`Note that importing other modules should occur after importing the tracer
> to ensure their supported dependencies are auto-instrumented.` 

be less confusing?

Reply:
I just realised that there is no need to import the tracer in this file. Tracing still is auto-instrumented. Hope I'm not overlooking something?

Reply:
I'd say just take it out